Pocket/pocket-ios
# Premium/free settings row

## Summary
* added premium row in settings and init popup

## References 
* [IN-1049](https://getpocket.atlassian.net/browse/IN-1049)
* [IN-1047](https://getpocket.atlassian.net/browse/IN-1047)

## Implementation Details
* added new settings row
* added default view for premium subscription

## Test Steps
* Be free user
* go to settings
* see row item for premium subscription
* clicking it will show a pop up for premium subscription
* swipe down AND top right "X" should dismiss view

## PR Checklist:

## Screenshots


[IN-1049]: https://getpocket.atlassian.net/browse/IN-1049?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IN-1047]: https://getpocket.atlassian.net/browse/IN-1047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
